include/dump_offload.hpp

```diff
@@ -442,7 +442,7 @@ inline void requestRoutes(App& app)
         }
 
         std::string url(conn.req.target());
-
+        BMCWEB_LOG_DEBUG << "Request open  for offload of system dump with url: "<<url;
```

Comment:
BMCWEB_LOG_DEBUG logs won't be logged into journal 
Use  BMCWEB_LOG_CRITICAL instead

Reply:
done